neon-mmd/websurfx
# ✨ Animations/effects support for the search engine

## What does this PR do?

This PR provides the default animation `simple-frosted-glow`, a new config option to change or disable the animations and a new settings option in the UI to change or disable the animations on the search engine website.

## Why is this change important?

This change is essential as it provides more customizability to the user, makes the search engine more fun to use and improves the user experience. 

## How to test this PR locally?

It can be tested by installing and running `Websurfx` as mentioned in the `docs` and on the `readme` and by launching the browser and thoroughly testing. By selecting the default animation `simple-frosted-glow` from the config and the settings page and check if the selected animation actually work in the search page of the website.

## Author's checklist

- [x] Provide a default animation `simple frosted glow`.
- [x] Provide a new config option for the animations
- [x] Provide a new settings option in the UI for the animations
- [x] Bump the app version to `1.6.0`   

## Related issues

Closes #424
